src/primitives/arc/mod.rs

```diff
@@ -91,7 +91,7 @@ impl Arc {
     /// Creates an arc based on a circle.
     ///
     /// The resulting arc will match the `top_left` and `diameter` of the base circle.
-    pub fn from_circle(circle: Circle, angle_start: Angle, angle_sweep: Angle) -> Self {
+    pub const fn from_circle(circle: Circle, angle_start: Angle, angle_sweep: Angle) -> Self {
```

Comment:
`Sector` contains the same constructor, which you missed.

Reply:
Thanks, added.